src/pages/User/components/UserData/styles.scss

```diff
@@ -0,0 +1,43 @@
+@use 'src/css/colors.variables.scss';
```

Comment:
This line of code is giving me an error when I try to run storybook, please check

![imagen](https://user-images.githubusercontent.com/66505715/189196617-2c0b8364-78e4-4544-94f1-a2007e3d5f60.png)